lib/r10k/action/runner.rb

```diff
@@ -42,23 +42,17 @@ def setup_logging
       def setup_settings
         config_settings = settings_from_config(@opts[:config])
 
-        overrides = {
-          cachedir: @opts[:cachedir],
-          deploy: {
-            puppet_path: @opts[:'puppet-path'],
-            generate_types: @opts[:'generate-types']
-          }
-        }
+        overrides = if @opts[:'puppet-path'].nil? && @opts[:'generate-types'].nil?
```

Comment:
Stylistically, we don't usually assign a value from a multiline if else.

Maybe either setting `overrides` in each branch, or something like:
```rb
overrides = {}
overrides[:deploy] = {} if @opts['puppet-path'] || @opts['generate-types']
```

Reply:
Fixed